otx/cli/builder/supported_backbone/mmcls.json

```diff
@@ -1,5 +1,154 @@
 {
   "backbones": {
+    "mmcls.AlexNet": {
+      "required": [],
+      "options": {},
+      "available": []
+    },
+    "mmcls.Conformer": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["tiny", "small", "base"]
+      },
+      "available": []
+    },
+    "mmcls.ConvMixer": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["768/32", "1024/20", "1536/20"]
+      },
+      "available": []
+    },
+    "mmcls.ConvNeXt": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["tiny", "small", "base", "large", "xlarge"]
+      },
+      "available": []
+    },
+    "mmcls.CSPDarkNet": {
+      "required": ["depth"],
+      "options": {
+        "depth": [53]
+      },
+      "available": []
+    },
+    "mmcls.CSPResNet": {
+      "required": ["depth"],
+      "options": {
+        "depth": [50]
+      },
+      "available": []
+    },
+    "mmcls.CSPResNeXt": {
+      "required": ["depth"],
+      "options": {
+        "depth": [50]
+      },
+      "available": []
+    },
+    "mmcls.DistilledVisionTransformer": {
+      "required": [],
+      "options": {},
+      "available": []
+    },
+    "mmcls.DenseNet": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["121", "169", "201", "161"]
+      },
+      "available": []
+    },
+    "mmcls.EfficientFormer": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["l1", "l3", "l7"]
+      },
+      "available": []
+    },
+    "mmcls.EfficientNet": {
+      "required": ["arch"],
+      "options": {
+        "arch": [
+          "b0",
+          "b1",
+          "b2",
+          "b3",
+          "b4",
+          "b5",
+          "b6",
+          "b7",
+          "b8",
+          "es",
+          "em",
+          "el"
+        ]
+      },
+      "available": []
+    },
+    "mmcls.HorNet": {
+      "required": ["arch"],
+      "options": {
+        "arch": [
+          "tiny",
+          "tiny-gf",
+          "small",
+          "small-gf",
+          "base",
+          "base-gf",
+          "base-gf384",
+          "large",
+          "large-gf",
+          "large-gf384"
+        ]
+      },
+      "available": []
+    },
+    "mmcls.HRNet": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["w32", "w18", "w30", "w40", "w44", "w48", "w64"]
+      },
+      "available": []
+    },
+    "mmcls.LeNet5": {
+      "required": [],
+      "options": {},
+      "available": []
+    },
+    "mmcls.MlpMixer": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["base", "small", "large"]
```

Comment:
I think "base" would be larger than "small"

Reply:
base is default value of arch